scripts/lint-changelogs.js

```diff
@@ -0,0 +1,66 @@
+/* global __dirname process */
+/* eslint-disable no-console */
+const {join, resolve} = require('path');
+
+const {readFileSync} = require('fs-extra');
+const glob = require('glob');
+
+const ROOT_PATH = resolve(__dirname, '..');
+
+const changelogs = readChangelogs();
+console.log(`🕵️‍♂️  Checking ${changelogs.length} changelogs`);
+
+const haveUnreleasedHeader = changelogs.filter(({packageChangelog}) =>
```

Comment:
this check is pretty naive but it works pretty well to insure folks at least go fix the ## Unreleased header on packages they are trying to release